airbyte-integrations/connectors/destination-databricks/src/main/kotlin/io/airbyte/integrations/destination/databricks/ConnectorClientsFactory.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.databricks
+
+import com.databricks.client.jdbc.Driver
+import com.databricks.sdk.WorkspaceClient
+import com.databricks.sdk.core.DatabricksConfig
+import io.airbyte.integrations.destination.databricks.model.Authentication
+import io.airbyte.integrations.destination.databricks.model.BasicAuthentication
+import io.airbyte.integrations.destination.databricks.model.DatabricksConnectorConfig
+import io.airbyte.integrations.destination.databricks.model.OAuth2Authentication
+import javax.sql.DataSource
+
+object ConnectorClientsFactory {
```

Comment:
what do we think about always prefixing our connector-specific classes with the connector name (`DatabricksConnectorClientFactory` here) ?